compatibility_server/compatibility_checker_server.py

```diff
@@ -82,16 +82,45 @@
 import wsgiref.simple_server
 
 import pip_checker
+import views
```

Comment:
Where does this module come from? The docker run fails at this line...

Reply:
Oh I see, you will need to add another like in Dockerfile `ADD views.py /compatibility_checker` to add that file into the docker image.

And could you test the docker build and docker run locally? The commands are in the README.rst file: https://github.com/GoogleCloudPlatform/cloud-opensource-python/tree/master/compatibility_server#commands-for-deployment

Reply:
Ok I will submit a new PR to address this.

However, it also looks like there is a gap in our testing I got all tests passing when I merged.

Reply:
That tests didn't include the docker build and run steps, which is done manually in each deployment. We could add that in the future, but this requires changing our CircleCI to run in a machine instead of running based on a docker... which won't be able to build a inner docker image easily.